pkg/kubelet/eviction/helpers.go

```diff
@@ -771,9 +771,9 @@ func debugLogObservations(logPrefix string, observations signalObservations) {
 	}
 	for k, v := range observations {
 		if !v.time.IsZero() {
-			klog.InfoS("Eviction manager:", "log", logPrefix, "signal", k, "resourceName", v.available, "capacity", v.capacity, "time", v.time)
+			klog.InfoS("Eviction manager:", "log", logPrefix, "signal", k, "available", v.available, "capacity", v.capacity, "time", v.time)
 		} else {
-			klog.InfoS("Eviction manager:", "log", logPrefix, "signal", k, "resourceName", v.available, "capacity", v.capacity)
+			klog.InfoS("Eviction manager:", "log", logPrefix, "signal", k, "available", v.available, "capacity", v.capacity)
```

Comment:
These changes appear to be unrelated, can you exclude them?

Reply:
Sure to get it landed sooner - will open another PR with the log update at some point bc it's confusing when debugging